app/bundles/CampaignBundle/Command/ExportCampaignCommand.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
+namespace Mautic\CampaignBundle\Command;
+
+use Mautic\CoreBundle\Command\ModeratedCommand;
+use Mautic\CoreBundle\Helper\CoreParametersHelper;
+use Mautic\CoreBundle\Helper\PathsHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ExportCampaignCommand extends ModeratedCommand
+{
+    use WriteCountTrait;
+
+    public function __construct(
+        PathsHelper $pathsHelper,
+        CoreParametersHelper $coreParametersHelper,
+    ) {
+        parent::__construct($pathsHelper, $coreParametersHelper);
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('mautic:campaign:export')
```

Comment:
As we discussed a couple of times, it should be able to export any entity
```suggestion
            ->setName('mautic:entity:export')
```

It would be nice to call it like `bin/console mautic:entity:export campaign 12`

Reply:
fixed